.travis.yml

```diff
@@ -116,8 +116,12 @@ script:
   # which should be investigated further in the future.
   - >
     if [[ $TEST_TARGET == 'doctest' ]]; then
+      MPL_RC_DIR=$HOME/.config/matplotlib;
+      mkdir -p $MPL_RC_DIR;
+      echo 'backend : Qt4Agg' > $MPL_RC_DIR/matplotlibrc;
```

Comment:
This should be ``agg``, right?

Reply:
If we only need to save images, then yes!